### ISSUE TITLE
Atom Freezes (becomes unresponsive) with large text file

### ISSUE THREAD

**Issue by wizardconsulting** [atom/atom#7777]
I installed Atom on a Windows 7 PC with 16gb of RAM (64-bit OS) and attempted to open up a 8mb text file.

The program stalled and could not open the file.

I was able to open up the same document using Notepad++ without issue.

I have, for the moment, removed the program until this issue is resolved.

Thanks,
--
Michael Frankel
President & CEO, Wizard Consulting Group, Inc.

**Comment by izuzak** [comment]
@wizardconsulting Would you be able to share that file? If not, can you share more information about it (e.g. what type of file was it -- was it a program in a specific language, and did it have a very long line)?

**Comment by mnquintana** [comment]
Also what version of Atom are you using?

**Comment by wizardconsulting** [comment]
Hi - 

The problem existed with Atom 1.0 for Windows (this was on Windows 7 64-bit). As of version 1.02, which I just installed this morning, the problem seems to have been resolved and/or eliminated.

Thanks,
-- 
Michael Frankel
Wizard Consulting Group, Inc.
FileMaker Business Alliance Member & Certified FileMaker Developer
Office: 818-706-8877
Mobile: 310-291-3419
Website: www.wizardconsultinggroup.com
